src/shared/components/palette/datetime-picker/datetime-picker-component.spec.ts

```diff
@@ -2,11 +2,9 @@ import { By } from '@angular/platform-browser';
 import { ComponentFixture, discardPeriodicTasks, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { NgxMatDatetimePickerModule, NgxMatNativeDateModule, NgxMatTimepickerModule }
-  from '@angular-material-components/datetime-picker';
+  from '@angular-material-components/datetime-picker/esm5/angular-material-components-datetime-picker.js';
```

Comment:
why is this /esm5/angular-material-components-datetime-picker.js needed?

Reply:
we need to use the es5 version of this npm package, otherwise IE 11 breaks. i also had to remove the moment-adapter as it doesnt have an es5 version we can use. Although i hate this approach, its the only way in such a short amount of time to keep the datepicker and date components within the toolkit

Reply:
ok